src/main/java/uk/gov/justice/digital/common/Util.java

```diff
@@ -0,0 +1,22 @@
+package uk.gov.justice.digital.common;
+
+import org.apache.spark.sql.Dataset;
+import org.apache.spark.sql.Row;
+
+import java.io.PrintWriter;
+import java.io.StringWriter;
+import java.util.Arrays;
+
+public class Util {
+
+    protected void handleError(final Exception e) {
+        final StringWriter sw = new StringWriter();
+        final PrintWriter pw = new PrintWriter(sw);
+        e.printStackTrace(pw);
+        System.err.print(sw.getBuffer().toString());
+    }
+
+    protected boolean schemaContains(final Dataset<Row> dataFrame, final String field) {
```

Comment:
This method only seems to be used in the domain executor so for now at least, it's probably best to define it there instead.